config/config.go

```diff
@@ -84,6 +84,9 @@ type Config struct {
 	// Use at your own risk or for unit testing purposes.
 	InsecureSkipVerify bool `name:"skip_verify" auth:"-"`
 
+	// marker for testing fixture
+	IsTesting bool
```

Comment:
can we make it private and move down to the list of arguments? and enable testing with `func (c *Config) WithTesting() *Config { c.isTesting = true; return c; }`?